src/caret_analyze/record/records_service/response_time.py

```diff
@@ -450,33 +450,46 @@ def __init__(
         self._histogram = ResponseHistogram(self._records, self._timeseries)
 
     def to_all_records(self) -> RecordsInterface:
-        return self._response_map_all.to_all_records()
+        """
+        Calculate the all records data for response time.
 
-    def to_worst_in_input_records(self) -> RecordsInterface:
-        return self._response_map_all.to_worst_in_input_records()
+        The all for response time are included message flow latency.
```

Comment:
This sentence feels unclear in its expression and is difficult to understand; please consider using a different way to convey this sentence.

Reply:
Fixed it in [d9d1ce7](https://github.com/tier4/CARET_analyze/pull/355/commits/d9d1ce7accdc922dfe6e445c6623d3c23746ec5d).